pkgs/tools/networking/curl/default.nix

```
@@ -196,6 +196,7 @@ stdenv.mkDerivation (finalAttrs: {
       # nginx-http3 = useThisCurl nixosTests.nginx-http3;
       nginx-http3 = nixosTests.nginx-http3;
       pkg-config = testers.testMetaPkgConfig finalAttrs.finalPackage;
+    } // lib.optionalAttrs (!stdenv.isDarwin && !stdenv.isx86_64) {
```

Is this right?  It removes the test on all Darwin platforms, and all x86_64 platforms.  Don't we want this?

```suggestion
    } // lib.optionalAttrs (!(stdenv.isDarwin && stdenv.isx86_64)) {
```

or the perhaps more difficult to get wrong

```suggestion
    } // lib.optionalAttrs (stdenv.hostPlatform.system != "x86_64-darwin") {
```

---

Boolean algebra is not easy :smile: .